keps/sig-storage/2639-secret-configmap-protection/kep.yaml

```diff
@@ -0,0 +1,38 @@
+title: Secret/ConfigMap Protection
+kep-number: 2639
+authors:
+  - "@mkimuram"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+status: implementable
+creation-date: 2021-04-20
+reviewers:
+  - "@liggitt"
+  - "@xing-yang"
+approvers:
+  - "@xing-yang"
+  - "@saad-ali"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.23"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.23"
+  beta: "v1.24"
+  stable: "v1.26"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: InUseProtection
```

Comment:
Please clarify that this feature gate is part of the "liens" KEP, not this KEP.

Reply:
@wojtek-t @xing-yang Are there good ways to describe above in kep.yaml? I've grep-ed other key.yaml, but failed to find a good example.